perf/terraform/configs/remote/terraform_override.tf

```
@@ -0,0 +1,20 @@
+terraform {
+  # TODO: Uncomment to start using remote backend
+  # backend "s3" {
+  #   bucket         = "terraform-tfstate"
```

I just checked and we already have a bucket with the name `libp2p-terraform-state` in the libp2p AWS account. What is the best practice? A single bucket with many `.tfstate` files differentiated by `key`, or multiple buckets, one per project, e.g. in this case one separate bucket for the libp2p-perf project.

---

Personally, if you already have `libp2p-terraform-state` bucket, I'd use this one. I've done bucket per project in some places at PL and it feels like an unnecessary overhead.

```suggestion
  backend "s3" {
    bucket         = "libp2p-terraform-state"
    key            = "github.com/libp2p/test-plans/perf/terraform/configs/remote/terraform.tfstate"
    region         = "us-west-2"
  }
}
```

Could you check what's the region of the bucket?

---

> Could you check what's the region of the bucket?

`us-west-2` if I recall correctly.

---

Cheers :) I uncommented the bucket name now - https://github.com/libp2p/test-plans/pull/185/commits/4179aa67c5836d880d88b000ec254d60cfdfc38c